js/client-battle-tooltips.js

```diff
@@ -213,7 +213,7 @@ var BattleTooltips = (function () {
 				text += '' + (move.desc || move.shortDesc) + '</p>';
 
 				if ('defrost' in move.flags) {
-					text += '<p>The user thaws out if it is frozen.</p>';
+					text += '<p class="movetag">The user thaws out if it is frozen.</p>';
```

Comment:
I have no idea, I was just making it consistent with the other lines. It's probably there for Stylish users or something.